gramps/plugins/db/bsddb/bsddb.gpr.py

```diff
@@ -17,6 +17,9 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
+from gramps.gen.const import GRAMPS_LOCALE as glocale
+_ = glocale.translation.gettext
```

Comment:
This line should come after the imports.  I normally move it together with its import.